src/main/java/com/blog/web/models/Article.java

```diff
@@ -0,0 +1,111 @@
+package com.blog.web.models;
+
+import com.blog.web.dto.ArticleDto;
+import jakarta.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class Article {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String photoUrl;
+    private String content;
+    @CreationTimestamp
+    private LocalDateTime createdOn;
+    @UpdateTimestamp
+    private LocalDateTime updatedOn;
+    @ManyToOne
+    @JoinColumn(name = "created_by", nullable = false)
+    private UserEntity createdBy;
+
+    public Article(
+            Long id,
+            String title,
+            String photoUrl,
+            String content,
+            UserEntity createdBy,
+            LocalDateTime createdOn,
+            LocalDateTime updatedOn
+    ) {
+        this.id = id;
+        this.title = title;
+        this.photoUrl = photoUrl;
+        this.content = content;
+        this.createdBy = createdBy;
+        this.createdOn = createdOn;
+        this.updatedOn = updatedOn;
+    }
+
+    public Article() {};
+
+    public Article(ArticleDto articleDto) {
+        this.id = articleDto.getId();
+        this.title = articleDto.getTitle();
+        this.photoUrl = articleDto.getPhotoUrl();
+        this.content = articleDto.getContent();
+        this.createdBy = articleDto.getCreatedBy();
+        this.createdOn = articleDto.getCreatedOn();
+        this.updatedOn = articleDto.getUpdatedOn();
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
do you want articles to have there id's changed without checking for duplicate id?